Meadowlark-js/packages/meadowlark-core/src/handler/DescriptorLoader.ts

```diff
@@ -101,12 +101,17 @@ async function loadParsedDescriptors(descriptorData: XmlDescriptorData): Promise
       }
 
       const descriptorDocument: DescriptorDocument = decapitalizeKeys(descriptorValue) as DescriptorDocument;
-      const descriptorEntityInfo: EntityInfo = {
+      const descriptorEntityInfo: DocumentInfo = {
         ...newEntityInfo(),
         entityName: descriptorName,
         projectName: 'Ed-Fi',
         projectVersion: '3.3.1-b',
-        naturalKey: buildNKString(`${descriptorDocument.namespace}#${descriptorDocument.codeValue}`),
+        documentIdentity: [
+          {
+            name: 'descriptor',
+            value: `${descriptorDocument.namespace}#${descriptorDocument.codeValue}`,
+          },
```

Comment:
Descriptor natural key strings were very different from entity natural key strings in Dynamo-centered Meadowlark. Shaping now as single DocumentElement with hardcoded name "descriptor" and value as before.

Reply:
I'm going to submit some comments before I lose them, and come back further review from this point onward tomorrow.